packages/@lwc/integration-karma/test/api/getComponentDef/index.spec.js

```diff
@@ -152,6 +153,26 @@ describe('@api', () => {
             childMethod: PublicMethodsInheritance.prototype.childMethod,
         });
     });
+
+    it('should log warning when accessing a private prop', () => {
+        const message = `Error: [LWC warn]: The property "privateGetter" is not publicly accessible. Add the @api annotation to the property declaration or getter/setter in the component to make it accessible.`;
+        const elm = createElement('x-private-accessor', { is: PrivateAccessors });
+        document.body.appendChild(elm);
+
+        expect(() => {
+            elm['privateGetter'];
+        }).toLogWarningDev(message);
+    });
+
+    it('should not log warning when accessing a public prop', () => {
+        const message = `Error: [LWC warn]: The property "publicGetter" is not publicly accessible. Add the @api annotation to the property declaration or getter/setter in the component to make it accessible.`;
+        const elm = createElement('x-private-accessor', { is: PrivateAccessors });
+        document.body.appendChild(elm);
+
+        expect(() => {
+            elm['publicGetter'];
+        }).not.toLogWarningDev(message);
```

Comment:
```suggestion
        }).not.toLogWarningDev();
```

No `message` necessary here; we just want to make sure it doesn't warn.